core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -178,6 +178,17 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
   private def clearBlacklist(expireTimeInSeconds: Long): Unit = {
     val expiredThreshold = clock.getTimeMillis() - expireTimeInSeconds * 1000
     blacklist.asScala.retain((_, creationTime) => creationTime >= expiredThreshold)
+    blacklist.asScala.retain { (fileName, creationTime) =>
```

Comment:
This can be merged into the line above

Reply:
Oops. Thanks. Yep. It's a copy&paste mistake. :(